app/views/schedule/index.html.erb

```diff
@@ -105,7 +105,7 @@
     <tr>
       <td></td>
       <td>
-        We're hoping this will be a collabrative activity.
+        We're hoping this will be a collaborative activity.
```

Comment:
This line has gone from master, could you rebase this PR please?

Reply:
will do :)